tests/integration/tier0/non-destructive/basic-sanity-checks/test_basic_sanity_checks.py

```diff
@@ -23,9 +23,7 @@ def test_check_user_privileges(shell):
     # Check the program exits as it is required to be run by root
     assert result.returncode != 0
     # Check the program exits for the correct reason
-    assert (
-        result.output == "The tool needs to be run under the root user.\n" "\n" "No changes were made to the system.\n"
-    )
+    assert "The tool needs to be run under the root user.\n" "\n" "No changes were made to the system." in result.output
```

Comment:
```suggestion
    assert "The tool needs to be run under the root user." "\n" "No changes were made to the system." in result.output
```
There's one extra newline which makes the validation fail.

Reply:
Thanks!